content/cloud-computing/concepts/types-of-cloud/types-of-cloud.md

```diff
@@ -0,0 +1,28 @@
+---
+Title: 'Types of Cloud'
+Description: 'There are three types of cloud computing implementations.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Cloud Computing'
+  - 'Networking'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+There are three **types** of cloud computing implementations:
+
+## Public Cloud
+
+Public clouds are owned and operated by third-party cloud service providers that deliver computing resources such as servers and storage over the Internet. Examples include Amazon Web Services (AWS), Google Cloud Platform (GCP), and Microsoft Azure.
+
+## Private Cloud
+
+Private cloud refers to cloud computing resources used exclusively by a single business or organization. A private cloud can be physically located on the company’s on-site data center. Some companies also pay third-party service providers to host their private cloud.
+
+## Hybrid Cloud
+
+Hybrid clouds are a combination of both private and public clouds. Data and applications are allowed to be shared between the private and public clouds. Allowing data and applications to move between private and public clouds provides businesses greater flexibility, more deployment options, and helps optimize existing infrastructure, security, and compliance.
+
+There is also what is referred to as **multicloud** implimentations, which combine the above. It refers to any implementation that utilizes more than one cloud solution. While it includes all hybrid clouds by definition, it also includes solutions that involve two or more public clouds, or two or more private clouds.
```

Comment:
Should have its own header. Also, the use of bold text is inconsistent in this article.

Reply:
Final sentence can remove "by definition"

Reply:
```suggestion
## Multicloud

Multicloud is a combination of the above, referring to any implementation that utilizes more than one cloud solution. While it includes all hybrid clouds, multicloud also includes solutions that involve two or more public clouds, or two or more private clouds.
```